game.html

```diff
@@ -23,5 +23,8 @@
     <div id="stats">Statistics</div>
     <div id="scores">Score: <span id="scoren">0</span></div>
     <div id="carryings">Carried Cakes: <span id="carryingn">0</span></div>
+    <div><button onClick="setfps(60)">60 fps</button> </div>
+    <div><button onClick="setfps(1000)">Max fps (for real gamerz)</button> </div>
```

Comment:
Not High enough
Can't you comprehend how many cakes Lex must acquire? As many as Four Tens I tell you!
He's in a rush!